lib/index.js

```diff
@@ -1,16 +1,24 @@
+//
+// Deprecated interface
+//
```

Comment:
do we have any documentation on this being deprecated? might want to add that too. Node generally does documentation -> runtime warning -> removal over time for their deprecation policy. IDK if we want to follow, but docs seem good either way.

Reply:
:+1: Sounds good.  I'm going to rewrite README.md next, so I'll start your suggested deprecation process with that re-write.